fearless/Modules/AccountManagement/AccountManagementPresenter.swift

```diff
@@ -8,56 +8,31 @@ final class AccountManagementPresenter {
     var wireframe: AccountManagementWireframeProtocol!
     var interactor: AccountManagementInteractorInputProtocol!
 
-    private var selectedAccountItem: AccountItem?
-
     let viewModelFactory: ManagedAccountViewModelFactoryProtocol
-    let supportedNetworks: [SNAddressType]
 
-    private var sections: [ManagedAccountViewModelSection] = []
+    private var viewModels: [ManagedAccountViewModelItem] = []
 
-    private let listCalculator: ListDifferenceCalculator<ManagedAccountItem> = {
-        let calculator = ListDifferenceCalculator<ManagedAccountItem>(initialItems: []) { item1, item2 in
+    private let listCalculator: ListDifferenceCalculator<ManagedMetaAccountModel> = {
```

Comment:
I would call it `PositionCalculator` or `ListPositionCalculator`

Reply:
Need a pr to Robinhood, but basically the whole point of the class if to find changes